src/components/AeInputAmount.vue

```diff
@@ -86,5 +112,9 @@ export default {
       color: $secondary_color;
     }
   }
+
+  .input-amount {
+    max-width: 95px;
+  }
```

Comment:
We are using rems throughout the project I guess this could be converted.

Reply:
the design is not perfected yet, maybe once we have some input from tsvetan we can work on this together to optimize :)